mailpoet/assets/js/src/newsletters/campaign-stats/newsletter-stats-info.tsx

```diff
@@ -38,9 +38,8 @@ const getEditorLink = (newsletter: NewsletterType) => {
     MailPoet.FeaturesController.isSupported('gutenberg_email_editor') &&
     newsletter.wp_post_id
   ) {
-    editorHref = `admin.php?page=mailpoet-email-editor&postId=${newsletter.wp_post_id}`;
+    editorHref = `post.php?post=${newsletter.wp_post_id}&action=edit`;
```

Comment:
We missed [this](https://github.com/mailpoet/mailpoet/blob/7bcfe93269b3532056cf5f6878aa70bff614af0d/mailpoet/assets/js/src/newsletters/send.tsx#L876-L879) check.  We might also use this opportunity to consider the initial option you raised of using a custom function for the Gutenberg email editor URL

Reply:
I would bet I was searching codebase for `mailpoet-email-editor`. Somehow I missed one. Thanks a lot for double checking. I added a function for the Gutenberg Editor URL in https://github.com/mailpoet/mailpoet/pull/5930/commits/912fe668233c26082f3bca4eb027f5dbefa54271